packages/frontend-2/pages/workspaces/[id]/index.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <WorkspaceDashboard v-if="isLoggedIn" />
+  <div v-else class="mx-auto">
+    <CommonLoadingIcon />
+  </div>
+</template>
+<script setup lang="ts">
+const { isLoggedIn } = useActiveUser()
+
+definePageMeta({
+  middleware: ['auth', 'requires-workspaces-enabled']
```

Comment:
The isLoggedIn check is not necessary if we have the auth middleware. Also - if this ends up being a page with a single component in it, why not just move WorkspaceDashboard code to here? Otherwise there's an unnecessary layer where nothing happens, just extra complication and jumping around the codebase. There are some older pages that do this, but only because we were trying to render them in storybook at one point